### DISCUSSION TITLE
CoPilot on PyCharm Issue

### DISCUSSION THREAD

**Discussion by avis1234** [community/community#7551] [Copilot Conversations] [answered]
"Show Previous completions" and "Show Next completions" do not seem to work. 
Actions are disabled even when the Copilot Solutions window presents multiple solutions.
Defining alternative shortcuts does not help.

**Comment by jansorg** [comment, marked as answer]
Which type of file were you editing? The alternative suggestions are currently only support in Python and Java files.

"Show next/previous" will be improved with the next update.

**Comment by avis1234** [comment]
Thanks for your reply.
I’m editing a Python file.
Opening the Co Pilot window within Pycharm shows the alternative
suggestions list but “Show next/previous” does not seem to work.
I’m using an international Mac keyboard (English / Hebrew) layout.

Best regards,
Avi
